scripts/update_apriori.py

```diff
@@ -10,11 +10,26 @@
 from hera_mc import mc, cm_utils, cm_partconnect
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     parser = mc.get_mc_argument_parser()
-    parser.add_argument('-p', '--hpn', help="HERA part number")
-    parser.add_argument('-s', '--status', help="New apriori status.",
-                        choices=['passed_checks', 'needs_checking', 'known_bad', 'not_connected'])
+    parser.add_argument("-p", "--hpn", help="HERA part number")
+    parser.add_argument(
+        "-s",
+        "--status",
+        help="New apriori status.",
+        choices=[
+            "dish_maintenance",
+            "dish_ok",
+            "RF_maintenance",
+            "RF_ok",
+            "digital_maintenance",
+            "digital_ok",
+            "passed_checks",
+            "needs_checking",
+            "known_bad",
+            "not_connected",
```

Comment:
Do we want to make the deprecated values easy to add via a script?

Reply:
In scripts/update_apriori.py, the old options are still there (I guess you get a deprecated message later), but the new versions aren't there.  Also, I think that we should include underscores in the new values.